server/src/main/java/org/elasticsearch/action/admin/indices/resolve/ResolveClusterActionRequest.java

```diff
@@ -202,6 +209,10 @@ boolean localIndicesPresent(String[] indices) {
         return false;
     }
 
+    public void setTimeout(String timeout) {
```

Comment:
Please don't expose this setter as a `String`. We should be converting to the proper type at the edge (i.e. in the REST handler).